docs/administration/security/storage-plugins/cyberark-storage.md

```diff
@@ -0,0 +1,102 @@
+# Cyberark Key Storage Plugin (Enterprise)
+
+Cyberark is a leader in Privileged Access Management solutions providing cloud ready, easy to use security solutions across the complete privileged access surface.  A centralized password management system provides visibility and control to protect privileges from attack.  Cyberark's solution is built for the Enterprise to enforce strong password business policies and prevent data breaches.
+
+Note: this storage plugin requires having the JRE installed, as well as the java keytool. In order to provide the correct authentication to the plugin, you will need to store all keys and certificates in key stores and specify them in `rundeck-config.properties`. For more information on creating/using a keystore, see [Java Keystore](https://www.ibm.com/docs/en/cognos-tm1/10.2.2?topic=ictocyoiatwas-add-certificates-jre-keystore). 
+
+:::: tabs
+::: tab Configuration Management
+
+Below is an example configuration, which can be configured using the *System Configuration* module. Add each setting as a configuration entry.
+
+![Cyberark Configuration](@assets/img/cyberark-config.png)
+
+![Cyberark Configuration - continued](@assets/img/cyberark-config2.png)
+
+:::
+::: tab rundeck-config.properties
+Alternatively the settings can be placed in `rundeck-config.properties`
+
+```
+rundeck.storage.provider.1.type=cyberark-storage
+rundeck.storage.provider.1.path=keys
+rundeck.storage.provider.1.removePathPrefix=true
+rundeck.storage.provider.1.config.baseURL=https://services-uscentral.skytap.com:18246
```

Comment:
Don't use "skytap".  Let's be a bit more explicit that it's a URL to "their Cyberark".  Maybe:

your.cyberarkserver.org 

For the port, is there a default/common port that Cyberark runs on?